src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -192,6 +193,11 @@ internal virtual HttpClient GetHttpClient(bool handleRedirect)
                 }
             }
 
+            // Set The SslProtocol. WebSslProtocol is used because not all SslProtocols are supported by HttpClientHandler.
+            // Also SslProtocols.Default is not the "default" for HttpClientHandler as SslProtocols.Ssl3 is not supported.
```

Comment:
I believe it is better to move the comment in WebSslProtocol definition and make it more accurate.

Reply:
will fix